service/storage_service.hh

```diff
@@ -291,13 +291,18 @@ private:
     future<> shutdown_protocol_servers();
 
     struct replacement_info {
+        locator::host_id replace_host_id;
```

Comment:
replace_host_id is a duplication of  host_id below.

Reply:
fixed in the next version